apps/meteor/app/api/server/v1/groups.ts

```diff
@@ -704,12 +709,9 @@ API.v1.addRoute(
 	{ authRequired: true },
 	{
 		async get() {
-			const findResult = await findPrivateGroupByIdOrName({
-				params: this.queryParams,
-				userId: this.userId,
-			});
+			const room = await getRoomFromParams(this.queryParams, { muted: 1, unmuted: 1 });
```

Comment:
Oops, this is dangerous :grimacing: 
`getRoomFromParams` misses some checks we have in `findPrivateGroupByIdOrName` (such as checking if the user has acces to the room or if it is archived). I'd suggest to keep using `findPrivateGroupByIdOrName`, but add an optional `roomFields` param to it so that you can send `{ muted: 1, unmuted: 1 }`
It should have just the same effect

Reply:
This made me find out we have no tests for trying to get members from a private group someone doesn't have access to :grimacing: